README.md

```diff
@@ -119,13 +119,19 @@ def get_next_audio_frame():
 speaker.write(get_next_audio_frame())
 ```
 
-When all frames have been written, run `stop()` on the instance:
+When all frames have been written, run `flush()` to wait for all buffered PCM data to be played:
+
+```python
+speaker.flush()
+```
+
+Once you are done writing PCM, run `stop()` on the instance:
 
 ```python
 speaker.stop()
 ```
 
-Once you are done, free the resources acquired by PvSpeaker. You do not have to call `stop()` before `delete()`:
+Once you are done completely, free the resources acquired by PvSpeaker. You do not have to call `stop()` before `delete()`:
```

Comment:
what happens if you call delete while the playback is still going? `completely` should be elaborated

Reply:
clarified that it should be called after `flush` or `stop_flush`